### PR TITLE
Various small documentation related changes

### PR DESCRIPTION
**Pull request by hhorak**
Use maintainer label consistently
Few edits in top-level README
Using usage label and command
Remove tables in README, because go-md2man cannot convert them properly

### CODE REVIEW

**Comment by hhorak** [comment]
Also removes release from CentOS and RHEL Dockerfiles.

File: 3.2/root/usr/bin/usage [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cat /usr/share/container-scripts/redis/README.md
```
Comment (omron93) [inline 3.2/root/usr/bin/usage]: Some other images uses  `groff` is usage script

https://github.com/sclorg/mariadb-container/blob/master/s2i-common/bin/usage

Comment (hhorak) [reply 3.2/root/usr/bin/usage]: Yes, although I'm not sure it should be a preferred way -- that would mean installing at least `groff-base` package more, and then showing the man page file is basically converting from markdown, to groff, then render it to plain text... which might or might not work as expected. Thinking about it more, simply showing README.md might be safer way..

Comment (omron93) [reply 3.2/root/usr/bin/usage]: Make sense...

**Comment by omron93** [comment]
Thanks for this PR.

**Pull request merged**